node_modules/.bin/acorn

```diff
@@ -1 +0,0 @@
-../acorn/bin/acorn
```

Comment:
Why is there almost 2000 removals of node modules from outside of the src folder

Reply:
I had to remove it for clarity! It was unneeded so thought this might have been the best approach